sacred/host_info.py

```diff
@@ -15,15 +16,22 @@
 
 __all__ = ('host_info_gatherers', 'get_host_info', 'host_info_getter')
 
-host_info_gatherers = {}
-"""Global dict of functions that are used to collect the host information."""
-
 
 class IgnoreHostInfo(Exception):
     """Used by host_info_getters to signal that this cannot be gathered."""
 
 
-def get_host_info():
+def check_additional_gatherers(additional_gatherers: dict):
+    for key in additional_gatherers:
+        if key in host_info_gatherers:
+            error_msg = (
+                'Key {} used in `additional_gatherers` already exists in for '
```

Comment:
I would write `"used in `additional_gatherers` already exists as a default gatherer function."`